lib/boat.rb

```diff
@@ -0,0 +1,21 @@
+class Boat
+  attr_reader :type, :cost
+
+  def initialize(type, cost)
+    @type = type
+    @cost = cost
+    @hours = 0
+  end
+
+  def price_per_hour
+    @cost
```

Comment:
You could alternatively name @cost @price_per_hour and put it in the attr_reader